### PR TITLE
small changes

### PR DESCRIPTION
**Pull request by stereobooster**
change window to w
GA now truely asynchronous
follow jshint rules
add min version

### CODE REVIEW

File: static/hn.js [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
-  var messageEvent = eventMethod == "attachEvent" ? "onmessage" : "message";
+  "use strict";
+  var j,
+      d = w.document,
+      hnAnchorElements = d.getElementsByClassName("hn-share-button"),
+      eventMethod = w.addEventListener ? "addEventListener" : "attachEvent",
+      eventer = w[eventMethod],
+      messageEvent = eventMethod === "attachEvent" ? "onmessage" : "message",
+      _gaq = w._gaq || [],
+      base = "ttp://hnbutton.appspot.com/";
```
Comment (igrigorik) [inline static/hn.js]: New protocol? ;)

**Comment by stereobooster** [comment]
change it in 2fecf96

File: static/hn.js [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
-  eventer(messageEvent,function(e) {
-    if (e.origin !== 'http://hnbutton.appspot.com/') { return; }
-    if (typeof(_gaq) != "undefined") {
-      if (e.data == 'vote' || e.data == 'submit') {
-        _gaq.push(['_trackSocial', 'Hacker News', e.data]);
-      }
+  eventer(messageEvent, function (e){
+    if (e.origin !== base) { return; }
+    if (e.data === "vote" || e.data === "submit") {
+      w._gaq = w._gaq || [];
```
Comment (mathiasbynens) [inline static/hn.js]: How about avoiding unneeded reassignments?

``` js
w._gaq || (w._gaq = []);
```

Comment (stereobooster) [inline static/hn.js]: done

**Comment by igrigorik** [comment]
Not sure why GH didn't autoclose.. merged in -- thanks!

**Pull request closed without merging**